### PR TITLE
Lw revamp link token creation

### PR DESCRIPTION
**Pull request by lindawoo-plaid**
1) changes the process for link token creation.  Instead of creating link token as soon as a new user is added, it waits until the enduser clicks "add a bank".

### CODE REVIEW

File: client/src/components/UpdateLink.tsx [hunk 1]
```diff
@@ -17,28 +14,32 @@ interface Props {
...
     setToken(linkTokens.byItem[props.itemId]);
   }, [linkTokens, props.itemId]);
 
   return (
     <div className="more-details">
-      {token != null && token.length > 0 && (
-        <LinkButton userId={props.userId} itemId={props.itemId} token={token}>
+      {
+        <Button small centered inline onClick={initiateLinkUpdate}>
+          {' '}
```
Comment (mattnguyen1) [inline client/src/components/UpdateLink.tsx:34]: is this something we could use CSS to create the spacing for instead?

File: client/src/components/UserCard.tsx [hunk 2]
```diff
@@ -74,12 +78,15 @@ export default function UserCard(props: Props) {
...
+              Add your checking or savings account
+            </Button>
+          )}
           {(props.removeButton || (props.linkButton && numOfItems === 0)) && (
             <div className="user-card__button">
               {token != null && token.length > 0 && props.linkButton && (
-                <LinkButton userId={props.userId} token={token} itemId={null}>
-                  Add your checking or savings account
-                </LinkButton>
+                <LinkButton userId={props.userId} token={token} itemId={null} />
```
Comment (mattnguyen1) [inline client/src/components/UserCard.tsx:89]: Is there supposed to be text in this button?

**Review by mattnguyen1 — APPROVED** [review]
lgtm, just some minor comments!

File: client/src/components/UserCard.tsx (hunk 2 above)
Comment (lindawoo-plaid) [reply client/src/components/UserCard.tsx:89]: no, no text in this button...I think I will do away with it being a button...

File: client/src/components/UpdateLink.tsx (hunk 1 above)
Comment (lindawoo-plaid) [reply client/src/components/UpdateLink.tsx:34]: deleted this...this was a prettier thing

File: client/src/components/UserCard.tsx (hunk 2 above)
Comment (lindawoo-plaid) [reply client/src/components/UserCard.tsx:89]: no, no text here. I think I will make this not a button any longer

**Pull request merged**